draft-ietf-jsonpath-base.md

```diff
@@ -612,118 +589,36 @@ Queries:
 | Query | Result | Result Path | Comment |
 | :---: | ------ | :----------: | ------- |
 | `$` | `{"k": "v"}` | `$` | Root node |
-{: title="Root selector examples"}
-
-### Dot Selector
-
-#### Syntax
-{: unnumbered}
-
-A dot selector starts with a dot `.` followed by an object's member name.
-
-~~~~ abnf
-dot-selector    = "." dot-member-name
-dot-member-name = name-first *name-char
-name-first      =
-                      ALPHA /
-                      "_"   /           ; _
-                      %x80-10FFFF       ; any non-ASCII Unicode character
-name-char = DIGIT / name-first
-
-DIGIT           =  %x30-39              ; 0-9
-ALPHA           =  %x41-5A / %x61-7A    ; A-Z / a-z
-~~~~
-
-Member names containing characters other than allowed by
-`dot-selector` — such as space (U+0020), minus (U+002D), or dot (U+002E)
-characters — MUST NOT be used with the `dot-selector`.
-(Such member names can be addressed by the
-`index-selector` instead.)
-
-#### Semantics
-{: unnumbered}
-
-The `dot-selector` selects the node of the member value corresponding
-to the member name from any JSON object in its input nodelist.
-
-It selects no nodes from any other JSON value.
-
-#### Examples
-{: unnumbered}
-
-JSON:
-
-    {"j": {"k": 3}}
-
-Queries:
-
-| Query | Result | Result Paths | Comment |
-| :---: | ------ | :----------: | ------- |
-| `$.j`   | `{"k": 3}` | `$['j']`      | Named value of an object      |
-| `$.j.k` | `3`        | `$['j']['k']` | Named value in nested object  |
-{: title="Dot selector examples"}
-
-### Dot Wildcard Selector {#wildcard}
-
-#### Syntax
-{: unnumbered}
+{: title="Root identifier examples"}
 
-The dot wildcard selector has the form `.*` as defined in the
-following syntax:
-
-~~~~ abnf
-dot-wild-selector    = "." wildcard       ;  dot followed by asterisk
-wildcard             = "*"
-~~~~
+## Selectors
 
-#### Semantics
-{: unnumbered}
+Selectors are defined solely within the context of the [child appender](#child-appender).
+A selector outside of a child appender is invalid.
 
-A `dot-wild-selector` acts as a wildcard by selecting the nodes of
-all member values of an object in its input nodelist as well as all
-element nodes of an array in its input nodelist.
+A selector identifies, or matches, child elements to be selected for the
+resulting nodelist.
```

Comment:
This is confusing. We should reserve identification language for `$` (and perhaps `@`). I think the rest of the PR suggests that selectors match rather than select and it's the child/descendant appender that actually does the selecting.